README.md

```diff
@@ -297,15 +297,24 @@ idioms as same-domain requests.
 
 ### Proxy
 
-You can use a proxy (e.g. [hapi-rest-proxy](https://github.com/chrishelgert/hapi-rest-proxy))
+You can use a proxy (e.g. [hapi-rest-proxy](https://github.com/chrishelgert/hapi-rest-proxy) or [cors-anywhere](https://github.com/Rob--W/cors-anywhere))
```

Comment:
I'm not sure if an example is not enough. There are 1000 different solutions (`nginx`, `traefix`, `cors-anywhere`, `cors`, ...).